### ISSUE TITLE
Forbid accessing this.args in class-body space

### ISSUE THREAD

**Issue by NullVoxPopuli** [ember-cli/eslint-plugin-ember#2347]
bad:
```js
class extends Component {
  // non-reactive!
  foo = this.args.foo;
}
```

good:
```js
class extends Component {
  // lazy, reactive
  get foo() {
    return this.args.foo;
  }
}

```

**Comment by kategengler** [comment]
That first case is useful if you understand that it isn't reactive

**Comment by NullVoxPopuli** [comment]
true
